hushline/admin.py

```diff
@@ -27,6 +30,13 @@ def toggle_admin(user_id: int) -> Response:
         user = db.session.get(User, user_id)
         if user is None:
             abort(404)
+
+        if user.is_admin:
```

Comment:
TOCTOU error with race condition here. This should be done in an explicit transaction.